src/shared/components/hero-section/index.js

```diff
@@ -145,14 +127,10 @@ class Hero extends Component {
     } else {
       errorMessage += error.message
     }
+    console.log(errorMessage)
```

Comment:
Remove this `console.log`

Reply:
This `console.log` was intended to be here. But I'll replace it using the toaster as you suggested.